.circleci/config.yml

```diff
@@ -22,8 +24,8 @@ jobs:
       - run:
           name : Run Unit Tests
           command: |
-            python3 -m unittest tests/test_linting.py
-            flake8 --max-line-length=120 --ignore=PT009
+            pytest --disable-warnings
+            flake8 --max-line-length=120 --ignore=PT009,E203,W503,W601
 
```

Comment:
Please add in the calls needed for unit testing on Windows/Linux (we don't have the right account for MacOS testing).  You can use the "integration" calls as a template.